docs/sources/view-and-analyze-profile-data/profile-cli.md

```diff
@@ -224,27 +224,27 @@ You can narrow the results down with the `--query` flag. See `profilecli help qu
 
 ### Reading a raw profile from a Pyroscope server
 
-You can use the `profilecli query merge` command to retrieve a merged (aggregated) profile from a Pyroscope server.
+You can use the `profilecli query profile` command to retrieve a merged (aggregated) profile from a Pyroscope server.
 The command merges all samples found in the profile store for the specified query and time range.
 By default it looks for samples within the last hour, though this can be controlled with the `--from` and `--to` flags. The source data can be narrowed down with the `--query` flag in the same way as with the `series` command.
 
-#### Query merge steps
+#### Query profile steps
 
 1. Specify optional flags.
 
    - You can provide a label selector using the `--query` flag, for example, `--query='{service_name="my_application_name"}'`.
    - You can provide a custom time range using the `--from` and `--to` flags, for example, `--from="now-3h" --to="now"`.
    - You can specify the profile type via the `--profile-type` flag. The available profile types are listed in the output of the `profilecli query series` command.
 
-2. Construct and execute the Query Merge command.
+2. Construct and execute the Query Profile command.
```

Comment:
Because we're referencing the command specifically, should we format it as a command?
```suggestion
2. Construct and execute the `query profile` command.
```